emulatedtests/utils/Config.json

```diff
@@ -0,0 +1,51 @@
+{
+    "UserConfig": {
+        "NamespaceConfig": [
```

Comment:
The EventHub names would most likely need to be changed for java tests, but if EventHub isn't being tested we don't need this file

Reply:
could be used for next pr to add eventhub